core/key.go

```diff
@@ -577,6 +625,46 @@ func (k CollectionSchemaVersionKey) ToDS() ds.Key {
 	return ds.NewKey(k.ToString())
 }
 
+func (k SchemaHistoryKey) ToString() string {
+	result := COLLECTION_SCHEMA_VERSION_HISTORY
+
+	if k.SchemaID != "" {
+		result = result + "/" + k.SchemaID
+	}
+
+	if k.PreviousSchemaVersionID != "" {
+		result = result + "/" + k.PreviousSchemaVersionID
+	}
+
+	return result
+}
+
+func (k SchemaHistoryKey) Bytes() []byte {
+	return []byte(k.ToString())
+}
+
+func (k SchemaHistoryKey) ToDS() ds.Key {
+	return ds.NewKey(k.ToString())
+}
+
+func (k SchemaVersionMigrationKey) ToString() string {
+	result := SCHEMA_MIGRATION
+
+	if k.SourceSchemaVersionID != "" {
+		result = result + "/" + k.SourceSchemaVersionID
+	}
+
+	return result
+}
+
+func (k SchemaVersionMigrationKey) Bytes() []byte {
```

Comment:
suggestion: would be nice to add a test for this like just to improve the test coverage